### PR TITLE
Modularization/fusion ieee arithmetic

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/arithmetic/regular/arithmetic.hpp [hunk 1]
```diff
@@ -12,17 +12,20 @@
 #include <eve/module/arithmetic/regular/absmin.hpp>
 #include <eve/module/arithmetic/regular/add.hpp>
 #include <eve/module/arithmetic/regular/agm.hpp>
+#include <eve/module/arithmetic/regular/arithmetic.hpp>
```
Comment (jfalcou) [inline include/eve/module/arithmetic/regular/arithmetic.hpp:15]: This is wrong

File: include/eve/module/arithmetic/detail/multi_div.hpp [hunk 2]
```diff
@@ -7,12 +7,11 @@
 //==================================================================================================
 #pragma once
 
+#include <eve/module/core.hpp>
 #include <eve/assert.hpp>
 #include <eve/function/div.hpp>
 #include <eve/function/mul.hpp>
 #include <eve/function/decorator.hpp>
-#include <eve/assert.hpp>
```
Comment (jfalcou) [inline include/eve/module/arithmetic/detail/multi_div.hpp:14]: Why was eve/assert.hpp removed ?

File: include/eve/module/arithmetic/regular/ieee.hpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
+//==================================================================================================
```
Comment (jfalcou) [inline include/eve/module/arithmetic/regular/ieee.hpp:1]: This file should not exist. 
If IEEE is in arithmetic, those thing should be in arithmetic.

File: include/eve/module/arithmetic/pedantic/ieee.hpp [hunk 4]
```diff
@@ -0,0 +1,16 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
```
Comment (jfalcou) [inline include/eve/module/arithmetic/pedantic/ieee.hpp:9]: This file should not exist. 
If IEEE is in arithmetic, those thing should be in arithmetic/pedantic.hpp.

File: include/eve/module/arithmetic/regular/impl/bit_floor.hpp [hunk 5]
```diff
@@ -8,18 +8,19 @@
...
+#include <eve/function/ifrexp.hpp>
+#include <eve/function/is_eqz.hpp>
 #include <eve/function/is_less.hpp>
-#include <eve/function/is_less_equal.hpp>
 #include <eve/function/ldexp.hpp>
-#include <eve/function/max.hpp>
-#include <eve/constant/one.hpp>
 #include <type_traits>
 #include <eve/concept/value.hpp>
-#include <eve/detail/apply_over.hpp>
```
Comment (jfalcou) [inline include/eve/module/arithmetic/regular/impl/bit_floor.hpp:22]: apply_over is still use, why remove the include ?

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/module/arithmetic/regular/arithmetic.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/arithmetic/regular/arithmetic.hpp:15]: done

File: include/eve/module/arithmetic/detail/multi_div.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/arithmetic/detail/multi_div.hpp:14]: reinserted

File: include/eve/module/arithmetic/pedantic/ieee.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/module/arithmetic/pedantic/ieee.hpp:9]: suppressed

File: include/eve/module/arithmetic/regular/impl/bit_floor.hpp (hunk 5 above)
Comment (jtlap) [reply include/eve/module/arithmetic/regular/impl/bit_floor.hpp:22]: reinserted

**Review by jfalcou — APPROVED** [review]

**Pull request merged**